scripts/build-deb.sh

```diff
@@ -13,7 +13,7 @@ ensureAvailable lintian
 ensureAvailable rpmbuild
 
 PACKAGE_TMPDIR=tmp/debian_pkg
-VERSION=`./artifacts/yarn-legacy-* --version`
+VERSION=`./artifacts/yarn-* --version`
```

Comment:
This won't work because `./artifacts/yarn-......tar.gz` also matches this glob: 
```
+ PACKAGE_TMPDIR=tmp/debian_pkg
++ ./artifacts/yarn-1.6.0.js ./artifacts/yarn-v1.6.0.tar.gz --version
 
error Command "./artifacts/yarn-v1.6.0.tar.gz" not found.
```

(https://circleci.com/gh/yarnpkg/yarn/14701?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)

Reply:
Thanks! Will see what we can do about it.